README.md

```diff
@@ -139,6 +139,8 @@ The setup script creates the following tables:
 - **cids:** Stores IPFS CIDs corresponding to bundles.
 - **balances:** Tracks token balances per vault.
 - **nonces:** Tracks the latest nonce for each vault.
```

Comment:
In a follow-up PR, we should track `nonces` in `vaults` instead of having a separate table.